.gitignore

```diff
@@ -3,8 +3,10 @@
 *~
 go
 t/servroot/
+t.stream/servroot/
```

Comment:
The C API for balancers is already merged into ngx_stream_lua_module's master. Let's change this.